### PR TITLE
Dockerizing

### PR DESCRIPTION
**Pull request by avsamit6600**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "appx-ui",
   "version": "0.1.0",
-  "proxy_server": "http://localhost:3000/",
```
Comment (dtadpole) [inline ui/package.json]: This change does not seem necessary, please kindly test functionality by removing this change

**Review by dtadpole — CHANGES_REQUESTED** [review]
Please pull latest code and update README file to include instructions on how to build and run with docker.  This can be a separate step in addition to original instruction of running the Dev environment

Thanks

File: ui/package.json (hunk 1 above)
Comment (avsamit6600) [reply ui/package.json]: Tested. Change unnecessary. Reverted

**Comment by avsamit6600** [comment]
@DracoTadpole Suggested change accepted. Relevant docs added to README

**Pull request merged**